Dockerfile_hugo

```diff
@@ -0,0 +1,11 @@
+FROM waterdata_labs_hugo_base
```

Comment:
Should this file named something else? I wasn't sure if there is a "best practice" for these?

Reply:
I would have done DockerfileLocal or something, but I don't think it's a big deal one way or the other. :man_shrugging: